mllib/src/main/scala/org/apache/spark/mllib/tree/loss/Loss.scala

```diff
@@ -49,4 +49,14 @@ trait Loss extends Serializable {
    */
   def computeError(model: TreeEnsembleModel, data: RDD[LabeledPoint]): Double
 
+  /**
+   * Method to calculate loss when the predictions are already known.
+   * Note: This method is used in the method evaluateEachIteration to avoid recomputing the
+   * predicted values from previously fit trees.
+   * @param datum: LabeledPoint
+   * @param prediction: Predicted label.
+   * @return Measure of model error on datapoint.
+   */
+  def computeError(datum: LabeledPoint, prediction: Double) : Double
```

Comment:
switch arg order to match batch computeError more closely

no space before colon (here and elsewhere)